spyder/widgets/mixins.py

```diff
@@ -1359,9 +1363,17 @@ def set_help_enabled(self, state):
         self.help_enabled = state
 
     def inspect_current_object(self):
-        text = self.get_current_word(help_req=True)
-        if text:
-            self.show_object_info(text, force=True)
+        regex_word = ''
+        start_text = self.get_text('sol', 'cursor')
+        tl1 = re.findall(r'([a-zA-Z_]+[^\s^\(]*)', start_text)

```

Comment:
Does this capture unicode variables? Please remember that unicode identifiers is valid in Python 3.

Reply:
Will check